### PR TITLE
Update faq for CHS

### LINKED ISSUE #1179: Updates to FAQ page for CHS merger
# TL;DR

 FAQ (https://lookit.mit.edu/faq/)
    `web/templates/web/faq.html`
   This page has lots of references to Lookit as the general site and/or platform, e.g.
   _"Here at **Lookit** we are very concerned about protecting your and your children's data"_
   _"**Lookit** supports recent versions of Chrome and Firefox."_
   _"If you are asking who runs this website, the answer is that we are a collaboration between two projects for online studies with children, **Lookit** and Children Helping Science. **Lookit** was founded by Kim Scott and Laura Schulz at MIT..."_ (...followed by more info about the history of Lookit and CHS. I don't think this section will be confusing for families but wanted to flag it for review).

# Narrative
What is the user story for this feature? the format should be as follows, e.g.:  "As a Researcher, I would like to have a new view to see all consented videos for a given experiment."

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
Leave important notes/suggestions here for implementation (important files, suggested approaches, etc.)

- Add "complaint" FAQ item for parents if they ahve a problem study

### PR DESCRIPTION
**Pull request by okaycj**
Update FAQ page to reflect changes needed for CHS merger.  

Closes #1179

### CODE REVIEW

File: web/templates/web/faq.html [hunk 1]
```diff
@@ -194,7 +195,7 @@ <h2 class="accordion-header" id="heading-9">
                  data-bs-parent="#faq-accordion-participation">
                 <div class="accordion-body">
                     {% blocktranslate %}
-                <p>Rather than having the parent or legal guardian sign a form, we ask that you read aloud (or sign in ASL) a statement of consent which is recorded using your webcam. This statement holds the same weight as a signed form, but should be less hassle for you. It also lets us verify that you understand written English and that you understand you're being videotaped.</p>
+                <p>Some studies ask you (the parent or guardian) to sign an online form after either reading about what the study involves or talking with a researcher. Other studies, especially ones that happen on our own Children Helping Science platform, ask that your read aloud (or sign in ASL) a statement of consent which is recorded using your webcam. This statement holds the same weight as a signed form, but should be less hassle for you. It also lets us verify that you understand written English and that you understand you're being videotaped.</p>
```
Comment (becky-gilbert) [inline web/templates/web/faq.html:198]: Seems like there's a typo here: "Other studies ... ask that your read aloud ... a statement of consent." I think "your" should be "you"? 
BTW I realize that this is the exact change that @mekline requested [here](https://github.com/lookit/lookit-api/issues/1179#issuecomment-1529937276) so I'm suggesting that the typo is from there.

File: web/templates/web/faq.html [hunk 2]
```diff
@@ -537,6 +538,34 @@ <h2 class="accordion-header" id="heading-20">
...
+                 data-bs-parent="#faq-accordion-participation">
+                <div class="accordion-body">
+                    <p>
+                        {% trans "Each study on Children Helping Science is run by a separate team of researchers. To find contact information for a specific study, visit your" %}
+                        <a href="{% url "web:studies-history" %}">Study History</a>
+                        {% trans "page. On this page you can also see information about when you participated in a study, and for studies that run here on our platform, watch your child's video from the session!" %}
+                    </p>
+                    <p>
+                        {% trans "If you have a more general question, want to report something to the Children Helping Science team, or need help reaching the researcher team, please contact us at" %}
+                        <a href="mailto:lookit-tech@mit.edu">lookit@mit.edu</a>!
```
Comment (becky-gilbert) [inline web/templates/web/faq.html:564]: Should the `mailto` address here be the same as the text, i.e. lookit@mit.edu?

File: web/templates/web/faq.html [hunk 3]
```diff
@@ -603,9 +631,8 @@ <h2 class="accordion-header" id="heading-23">
...
...rchers could go to connect with each other to support research into child development! The new Children Helping Science website is a collaboration between two previous platforms, Lookit and (an earlier version of) Children Helping Science. Lookit was founded by <a href = "https://kimberscott.github.io/">Kim Scott</a> and <a href = "http://eccl.mit.edu/">Laura Schulz</a> at MIT, and is now lead by Executive Director <a href = "http://www.melissaklinestruhl.com/">Melissa Kline Struhl</a>.  The original Children Helping Science website was created by <a href = "https://projects.iq.harvard.edu/ccdlab">Elizabeth Bonawitz</a> at Harvard, <a href = "http://sll.stanford.edu/">Hyowon Gweon</a> at Stanford, <a href = "https://compdevlab.yale.edu/">Julian Jara-Ettinger</a> at Yale, <a href = "https://www.utdallas.edu/thinklab/">Candice Mills</a> at UT Dallas, <a href = "http://eccl.mit.edu/">Laura Schulz</a> at MIT, and <a href = "https://www.minerva.kgi.edu/people/mark-sheskin-phd/">Mark Sheskin</a> at Minerva University. Mark did the web development (based on an initial design by Junyi Chu, a student in Laura's lab) and the logo was designed by Natalia Vélez, a student in Hyowon's lab. </p>
```
Comment (becky-gilbert) [inline web/templates/web/faq.html:635]: I think the [comment](https://github.com/lookit/lookit-api/issues/1179#issuecomment-1529951452) about this section from the FAQ issue also says to remove the last sentence in this paragraph.

File: web/templates/web/faq.html [hunk 4]
```diff
@@ -750,10 +777,15 @@ <h2 class="accordion-header" id="heading-29">
...
-                <p>As of January 2022, we have agreements with over 50 universities, including institutions in the United States, Canada, the United Kingdom, and European Union. Please email <a href="mailto:lookit-tech@mit.edu">lookit@mit.edu</a> if you are not sure whether your institution already has an agreement, or if you need any help at all getting your paperwork in order.</p>
-                <p>While you are completing these steps, you can get started on Lookit by creating a lab account, creating your first study, and getting it peer reviewed by our researcher community. A step-by-step guide to getting started is available <a href="https://lookit.readthedocs.io/en/develop/researchers-start-here.html">here</a>.</p>
+                <p>As of April 2023, we have agreements with over 90 universities, including institutions in the United States, Canada, the United Kingdom, and European Union. Please email <a href="mailto:lookit-tech@mit.edu">lookit@mit.edu</a> if you are not sure whether your institution already has an agreement, or if you need any help at all getting your paperwork in order.</p>
```
Comment (becky-gilbert) [inline web/templates/web/faq.html:787]: Same comment about the email address here: lookit@mit.edu vs lookit-tech@mit.edu. Should we use the same address?
(And I realize that this is how it was already, not something you changed!)

**Review by becky-gilbert — COMMENTED** [review]
Looks great! 👍 
I left a few comments/questions, some of which might be for @mekline to weigh in on.

File: web/templates/web/faq.html (hunk 2 above)
Comment (okaycj) [reply web/templates/web/faq.html:564]: Absolutely. I copied that a-tag from somewhere else on the page as well... so I'll have to find that too.

File: web/templates/web/faq.html (hunk 4 above)
Comment (okaycj) [reply web/templates/web/faq.html:787]: Yup you found it!  I don't know why it was like that, but I'll make the change.

File: web/templates/web/faq.html (hunk 3 above)
Comment (okaycj) [reply web/templates/web/faq.html:635]: The comment on the issue has two paragraphs.  The remaining text was removed.  Here's the comment from the issue and the FAQ in the local environment.  

Issue comment:
Screenshot 2023-05-11 at 9 58 33 AM

FAQ:
Screenshot 2023-05-11 at 9 59 13 AM

**Pull request merged**